extendr-api/Cargo.toml

```diff
@@ -34,6 +34,8 @@ rstest = "0.16.0"
 [features]
 default = []
 
+full-support = ["graphics", "either", "ndarray", "num-complex", "serde"]
```

Comment:
Choice of words matters here. I think we need to be 
```suggestion
# This dummy feature enables all features that increase the functionality of 
# extendr, via conversions or R features. Features that change behaviour
# but do not add functionality (such as `libR-sys/use-bindgen`) are excluded
full-functionality = ["graphics", "either", "ndarray", "num-complex", "serde"]
```

Reply:
Call it `all-extentions`?